sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1459,6 +1459,29 @@ def _path_residuals(
 class LinearModelCV(MultiOutputMixin, LinearModel, ABC):
     """Base class for iterative model fitting along a regularization path."""
 
+    _parameter_constraints = {
+        "eps": [Interval(Real, 0, None, closed="left")],
+        "n_alphas": [Interval(Integral, 0, None, closed="left")],
+        "alphas": [np.ndarray, None],
+        "fit_intercept": [bool],
+        "normalize": "no_validation",
+        "precompute": [StrOptions({"auto"}), "array-like", bool],
+        "max_iter": [Interval(Integral, 0, None, closed="left")],
```

Comment:
While technically `max_iter=0` is a valid input I don't think it makes much sense since it just guarantees no convergence and results in no change to the model. Indeed currently we set the minimum to 1, as in `ElasticNet._parameter_constraints`
```suggestion
        "max_iter": [Interval(Integral, 1, None, closed="left")],
```